src/@next/Badge/BadgeStyle.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from 'styled-components';
+import { S08 } from '../utilities/colors/blue';
+import { B89 } from '../utilities/colors/green';
+import { B95 } from '../utilities/colors/neutral';
+import { S42, S87 } from '../utilities/colors/orange';
+import { B100 } from '../utilities/colors/red';
+import { S75 } from '../utilities/colors/yellow';
+import { space8 } from '../utilities/spacing';
+import { BadgeProps, BadgeStatusType } from './Badge';
+
+const badgeBackgroundColor = {
+  success: B89,
```

Comment:
can you use the `color.codeName` format (example: `Blue.S08`) so it's easier to know which color they are?

Reply:
resolved